### ISSUE TITLE
Links don't appear to be clickable

### ISSUE THREAD

**Issue by MattRead** [agronick/Relay#29]
Links in chat don't look clickable because the cursor doesn't change and they are not underlined. When hovering a link the cursor should change to a hand and links be underlined.

**Comment by agronick** [comment]
Yeah, I've tried to do this. I can't find a way to change the cursor using the GTK apis. Maybe I just need to look into it more.

**Comment by agronick** [comment]
I spent a long time with this one. I tried code that would change the cursor on mouseover and change it back on mouseout but the methods don't seem to work.
